tests/subsys/bootloader/external_flash/src/app_update.h

```diff
@@ -0,0 +1,2471 @@
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xb4, 0x1d, 0x00, 0x20,
+  0xb4, 0x1d, 0x00, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x0f, 0x00, 0x00, 0x00, 0xc8, 0x1d, 0x00, 0x20, 0xc8, 0x1d, 0x00, 0x20,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x0f, 0x00, 0x00, 0x00,
+  0xdc, 0x1d, 0x00, 0x20, 0xdc, 0x1d, 0x00, 0x20, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x0f, 0x00, 0x00, 0x00, 0xf0, 0x1d, 0x00, 0x20,
+  0xf0, 0x1d, 0x00, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x0f, 0x00, 0x00, 0x00, 0x04, 0x1e, 0x00, 0x20, 0x04, 0x1e, 0x00, 0x20,
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x0f, 0x00, 0x00, 0x00,
+  0x18, 0x1e, 0x00, 0x20, 0x18, 0x1e, 0x00, 0x20, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x0f, 0x00, 0x00, 0x00, 0x07, 0x69, 0x50, 0x01,
+  0x10, 0x00, 0x20, 0x00, 0xff, 0x9a, 0x8f, 0xb3, 0xc9, 0xe7, 0x4e, 0x7f,
+  0x34, 0x9d, 0x7a, 0x8e, 0xb5, 0x4d, 0x19, 0x0b, 0x7b, 0xd6, 0x6b, 0x87,
+  0x69, 0x91, 0x46, 0xd3, 0x00, 0x16, 0x58, 0xf8, 0x39, 0x97, 0x4b, 0x21,
+  0x01, 0x00, 0x20, 0x00, 0xfc, 0x57, 0x01, 0xdc, 0x61, 0x35, 0xe1, 0x32,
+  0x38, 0x47, 0xbd, 0xc4, 0x0f, 0x04, 0xd2, 0xe5, 0xbe, 0xe5, 0x83, 0x3b,
+  0x23, 0xc2, 0x9f, 0x93, 0x59, 0x3d, 0x00, 0x01, 0x8c, 0xfa, 0x99, 0x94,
+  0x20, 0x00, 0x00, 0x01, 0xc5, 0x9b, 0xf3, 0x3e, 0xcc, 0x4c, 0xab, 0xf3,
+  0xb4, 0x29, 0xf5, 0x1b, 0x2c, 0xd8, 0xd5, 0xb7, 0xa9, 0x54, 0x19, 0x1e,
+  0xed, 0x83, 0x96, 0x43, 0xc1, 0xec, 0xfa, 0x59, 0x4e, 0x15, 0xe6, 0xa9,
+  0x0e, 0xc5, 0xf3, 0x0f, 0xeb, 0xc8, 0xea, 0xb8, 0x41, 0x65, 0xe2, 0xce,
+  0xee, 0xd7, 0x32, 0xa6, 0x41, 0xba, 0xe3, 0x61, 0x1e, 0x01, 0x57, 0x77,
+  0xc1, 0x03, 0x96, 0xf6, 0xe9, 0x36, 0xdb, 0x8d, 0xa7, 0xb2, 0x38, 0x4e,
+  0x6e, 0x63, 0x2b, 0x98, 0x8a, 0xe7, 0x1e, 0xc1, 0x30, 0xb3, 0xe1, 0xb7,
+  0x34, 0xa1, 0xf0, 0x57, 0xf1, 0x91, 0x3c, 0x7a, 0xcb, 0x7d, 0x37, 0xbe,
+  0xb0, 0x19, 0x70, 0xb3, 0xe3, 0xb4, 0x0a, 0x2d, 0x84, 0x6b, 0x71, 0xa2,
+  0x1e, 0xbe, 0xca, 0xdc, 0x2b, 0x44, 0x3c, 0xeb, 0xd8, 0x95, 0xdb, 0x0d,
+  0x05, 0x0e, 0xa9, 0x14, 0x57, 0x63, 0x44, 0xd6, 0x11, 0x65, 0xc0, 0x23,
+  0xf2, 0xb2, 0xb4, 0x73, 0xeb, 0xc2, 0xdd, 0x0c, 0xcd, 0x62, 0xcf, 0x56,
+  0xac, 0x9e, 0xb8, 0xb6, 0x9c, 0xc3, 0x5c, 0x5a, 0xdc, 0x7f, 0x18, 0x00,
+  0x9c, 0x1e, 0x3b, 0xd6, 0xff, 0xa2, 0x60, 0x82, 0x24, 0xbf, 0x9d, 0xf5,
+  0xd1, 0xf4, 0x52, 0x81, 0x57, 0x34, 0x52, 0x8f, 0xe6, 0xec, 0x45, 0xc3,
+  0x91, 0x4b, 0x37, 0x88, 0x87, 0xa0, 0x06, 0x55, 0xd3, 0x67, 0x91, 0x7f,
+  0xdb, 0x28, 0xaa, 0xb1, 0xe8, 0x93, 0x84, 0x56, 0xe8, 0xc4, 0x38, 0xc7,
+  0x1d, 0x44, 0x49, 0x7c, 0x8f, 0xd1, 0x8a, 0xc9, 0x2b, 0x39, 0x01, 0x69,
+  0xa9, 0x13, 0x17, 0x9f, 0x5e, 0x7a, 0x28, 0xee, 0xa9, 0xe7, 0x78, 0x6a,
+  0x3c, 0x1a, 0x85, 0x73, 0x97, 0x8d, 0xb8, 0xc5, 0xbe, 0x0a, 0x18, 0xbf,
+  0x6c, 0xf5, 0xf9, 0xcd, 0xe9, 0x0a, 0x16, 0xdc, 0x41, 0x50, 0x61, 0xff,
+  0x15, 0x58, 0x2b, 0x1d, 0xac, 0xc7, 0x50, 0xca
+};
+unsigned int app_update_bin_len = 29612;
```

Comment:
Why not do a `sizeof`? It's very easy to forget to update this.

Reply:
generated from `xxd`

Reply:
Ok, add a comment about that somewhere.